pinot-controller/src/test/java/org/apache/pinot/controller/api/resources/PinotSegmentUploadDownloadRestletResourceTest.java

```diff
@@ -56,20 +66,27 @@ public class PinotSegmentUploadDownloadRestletResourceTest {
 
   private static final String TABLE_NAME = "table_abc";
   private static final String SEGMENT_NAME = "segment_xyz";
+  private static final String HOST = "localhost";
+  private static final String PORT = "12345";
+  private static final File DATA_DIR =
+      new File(FileUtils.getTempDirectory(), "PinotSegmentUploadDownloadRestletResourceTest");
+  private static final File LOCAL_TEMP_DIR = new File(DATA_DIR, "localTemp");
```

Comment:
Delete these directories in `tearDown()` or see if you can use `_tempDir ` itself

Reply:
added logic in teardown